endpoint/common.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2019-2020 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#ifndef ENDPOINT_COMMON_H
+#define ENDPOINT_COMMON_H
+
+/**
+ * @file endpoint/common.h
+ * @brief Macros for endpoint 
+ */
+
+#define STR_HELPER(num) #num
```

Comment:
These macros should not be specific to endpoint. Instead, promote to global header.